server/market/market.go

```diff
@@ -178,20 +183,39 @@ func NewMarket(cfg *Config) (*Market, error) {
 		return nil, err
 	}
 
+	log.Infof("Allowing %d lots on the book per user.", mktInfo.BookedLotLimit)
+
 	// Load existing book orders from the DB.
 	base, quote := mktInfo.Base, mktInfo.Quote
+
 	bookOrders, err := storage.BookOrders(base, quote)
 	if err != nil {
 		return nil, err
 	}
-	log.Infof("Allowing %d lots on the book per user.", mktInfo.BookedLotLimit)
-
 	log.Infof("Loaded %d stored book orders.", len(bookOrders))
+
+	baseIsAcctBased := cfg.CoinLockerBase == nil
+	quoteIsAcctBased := cfg.CoinLockerQuote == nil
+
 	// Put the book orders in a map so orders that no longer have funding coins
 	// can be removed easily.
 	bookOrdersByID := make(map[order.OrderID]*order.LimitOrder, len(bookOrders))
-	for _, ord := range bookOrders {
-		bookOrdersByID[ord.ID()] = ord
+	for _, lo := range bookOrders {
+		// Limit order amount requirements are simple unlike market buys.
+		if lo.Quantity%mktInfo.LotSize != 0 || lo.FillAmt%mktInfo.LotSize != 0 {
+			// To change market configuration, the operator should suspended the
+			// market with persist=false, but that may not have happened, or
+			// maybe a revoke failed.
+			log.Errorf("Not rebooking order %v with amount (%v/%v) incompatible with current lot size (%v)",
+				lo.ID(), lo.FillAmt, lo.Quantity, mktInfo.LotSize)
+			// Revoke the order, but do not count this against the user.
```

Comment:
Thank you for adding this.  It needs to be pulled out for a 0.4 commit as well, and ideally would be a separate commit in this PR too.

Reply:
It was actually just moved from below.

Reply:
Ayyyy, it's been there since launch. Derp. https://github.com/decred/dcrdex/commit/6d80ad18914ff6803e2d3669edbd586ca6ad6dae